test/e2e/framework_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"gotest.tools/assert"
+	"testing"
+)
+
+func TestReplaceFieldValue(t *testing.T) {
```

Comment:
It's a bit strange that this will be run as part of the e2e tests, but I don't have a better idea and since it completes in an instant, it's not really an issue

Reply:
Yes, test code needs testing.

Reply:
I found this test when looking at several test failures and it happened to be the last succeeded e2e test in half cases. Although it didn't take me much time to exclude it as a suspect, it was distracting to debug a unit test in e2e test suite. Does it really help to test testing code? If the function is broken, won't the target test fail?

Reply:
I have an idea. We could add a build tag to this test file. The e2e test suite will no longer run the test, but we can run it as part of the regular Go unit testing Github workflow using a command like this one:

```
go test --tags=e2e-utest github.com/vmware-tanzu/antrea/test/e2e/...
```